app/controllers/challenge_claims_controller.rb

```diff
@@ -1,142 +1,99 @@
 class ChallengeClaimsController < ApplicationController
-
   before_filter :users_only
-  before_filter :load_collection, :except => [:index]
-  before_filter :collection_owners_only, :except => [:index, :show, :create, :destroy]
-  before_filter :load_claim_from_id, :only => [:show, :destroy]
-
-  before_filter :load_challenge, :except => [:index]
-  
-  before_filter :allowed_to_destroy, :only => [:destroy]
-
+  before_filter :load_collection, except: [:index]
+  before_filter :collection_owners_only, except: [:index, :show, :create, :destroy]
+  before_filter :load_claim_from_id, only: [:show, :destroy]
+  before_filter :load_challenge, except: [:index]
+  before_filter :allowed_to_destroy, only: [:destroy]
 
   # PERMISSIONS AND STATUS CHECKING
 
   def load_challenge
     if @collection
       @challenge = @collection.challenge
     elsif @challenge_claim
-      @challenge = @challenge_claim.collection.challenge
+      # TODO: Replace with &. operator after upgrading to Ruby 2.3
+      @challenge = @challenge_claim.collection.challenge if @challenge_claim.collection
     end
-    no_challenge and return unless @challenge
+    no_challenge && return unless @challenge
```

Comment:
I don't think `&& return` actually works, even though Hound says and our tests are passing.